### ISSUE TITLE
Positive or negative word

### ISSUE THREAD

**Issue by dwicak** [StarlangSoftware/TurkishSentiNet#2]
Hello, may I ask something about your sentiment lexicon?

`<WORD><NAME>avurtlamak</NAME><PSCORE>0.0</PSCORE><NSCORE>0.375</NSCORE></WORD>`

Is PSCORE is the positive word? and NSCORE is the negative word? And when both of them are 0.0, does it mean neutral word? Thank you very much.

**Comment by olcaytaner** [comment]
Yes you are right. If pscore is positive it is a positive word, if nscore
is positive it is a negative word, and if both of them are 0, it is a
neutral word.

9 Şub 2022 Çar 18:29 tarihinde darmawan wicaksono ***@***.***>
şunu yazdı:

**Comment by dwicak** [comment]
Thank you very much.
